deploy/05-operator.yaml

```diff
@@ -36,4 +36,12 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             - name: OPERATOR_NAME
-              value: "cluster-samples-operator"
+              value: cluster-samples-operator
+            - name: IMAGE_JENKINS
+              value: quay.io/openshift/origin-jenkins:latest
+            - name: IMAGE_AGENT_BASE
+              value: quay.io/openshift/origin-jenkins-agent-base:latest
+            - name: IMAGE_NODEJS_AGENT
+              value: quay.io/openshift/origin-jenkins-agent-nodejs:latest
+            - name: IMAGE_MAVEN_AGENT
```

Comment:
might as well fix the env var names too.

Reply:
fixed